src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1296,7 +1297,7 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
 
                                     $posmsg = ''
                                     if ($ErrorView -eq 'ConciseView') {
-                                        $posmsg = Get-ConciseViewPositionMessage
+                                        $posmsg = Get-ConciseViewPositionMessage -Err $_
```

Comment:
You should pass in `$err` instead to keep the original behavior.
I'd prefer to not changing this. `$myinv` used in `Get-ConciseViewPositionMessage` also comes from the parent scope, so just continue to read both `$err` and `$myinv` from the parent scope.

Reply:
That's fair.  Let me revert this change to keep it minimal.